src/provider/AzureDevops.ts

```diff
@@ -0,0 +1,42 @@
+import { Embed, EmbedField, EmbedAuthor } from '../model/DiscordApi'
+import { TypeParseProvder } from './BaseProvider'
+
+export class AzureDevops extends TypeParseProvder {
+
+    private embed: Embed
+
+    constructor() {
+        super()
+        this.setEmbedColor(0x205081)
+        this.embed = {}
+    }
+
+    public getName(): string {
+        return 'Azure-DevOps'
+    }
+
+    public getType(): string | null {
+        return this.body.eventType
+    }
+
+    public knownTypes(): string[] {
+        return ['gitPush']
+    }
+
+    public async gitPush(): Promise<void> {
+        this.logger.debug('Parsing git push event')
+        this.embed.title = this.body.message.markdown
```

Comment:
You can't use markdown in the title of an embed. You should use `this.body.message.text` instead.